operators/ibm-block-csi-operator-community/1.7.0/bundle-1.7.0.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator-community
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.8"
```

Comment:
@oriyarde you will skip this test by adding `=`, which means `4.8 only`.
```
LABEL com.redhat.openshift.versions="=v4.8"
```

Reply:
then why does the 4.9 test fails? is current label not allowed?

Reply:
Problem is that label `v4.8` means that it will be included in `v4.8` and `v4.9` and higher. If you want to limit specific one version you have to add `=` in front (eg. `=v4.8`). Or if you want range then you could `v4.6-v4.8`

Reply:
what if I'm OK with `v4.8` or higher? why does the 4.9 test fail?

Reply:
could I [remove `v4.9`](https://github.com/k8s-operatorhub/community-operators/pull/237) from the merged k8s bundle?